front/lib/api/enterprise_connection.ts

```diff
@@ -0,0 +1,83 @@
+import type { SupportedEnterpriseConnectionStrategies } from "@dust-tt/types";
+import type { Connection } from "auth0";
+import { ManagementClient } from "auth0";
+
+import config from "@app/lib/api/config";
+
+const management = new ManagementClient({
+  domain: config.getAuth0TenantUrl(),
+  clientId: config.getAuth0M2MClientId(),
+  clientSecret: config.getAuth0M2MClientSecret(),
+});
+
+function makeEnterpriseConnectionName(workspaceId: string) {
+  return `workspace-${workspaceId}`;
+}
+
+export async function getEnterpriseConnectionForWorkspace(
+  workspaceId: string,
```

Comment:
Let’s us auth here?

Reply:
And other functions in lib/api 👍

Reply:
I wish we had a way to narrow down the type of Authenticator sometimes to avoid all the logic to please Typescript 🙃.

Reply:
Fixed.